### ISSUE TITLE
AIMET1.25.0 keras QAT issue

### ISSUE THREAD

**Issue by wangjun-0** [quic/aimet#2013]
I try to use AIMET1.25.0 to quantize keras model by QAT, but I got the warning below:

WARNING:tensorflow:Model failed to serialize as JSON. Ignoring... maximum recursion depth exceeded
tensorflow - WARNING - Model failed to serialize as JSON. Ignoring... maximum recursion depth exceeded
Epoch 1/10
WARNING:tensorflow:Gradients do not exist for variables ['conv1_pad_input_quantizer_0.encoding_min:0', 'conv1_pad_input_quantizer_0.encoding_max:0', 'conv1_pad_output_quantizer_0.encoding_min:0', 'conv1_pad_output_quantizer_0.encoding_max:0', 'conv1_conv_input_quantizer_0.encoding_min:0', .....

After the warning, QAT is terminated.

How can I continue to do QAT?

**Comment by quic-akhobare** [comment]
Hi @wangjun-0 -  could you share the full stack trace?

**Comment by quic-mangal** [comment]
Closing this issue due to inactivity. Please create a new one if you are still stuck.
